server/src/main/java/org/opentosca/toscana/plugins/kubernetes/docker/dockerfile/builder/commands/RunCommand.java

```diff
@@ -0,0 +1,34 @@
+package org.opentosca.toscana.plugins.kubernetes.docker.dockerfile.builder.commands;
+
+import java.io.PrintWriter;
+
+public class RunCommand extends DockerfileEntry {
+    private final String command;
+
+    public RunCommand(String command) {
+        this.command = command;
+    }
+
+    @Override
+    public void appendToDockerfile(PrintWriter out,
```

Comment:
Inconsistent formatting. Is there something wrong with your settings?

Reply:
No. I just hate to scroll left and right if something overshoots my slightly non widescreen monitor.

Thats why i usualy define long methods like this:
```java
public void appendToDockerfile(
        PrintWriter out,
        DockerfileEntry previous,
        DockerfileEntry next,
        boolean compact
    ) {
```

Reply:
In general I have nothing against writing long constructors like this but the inconstancy between classes with the same method or constructor bother me.